src/main/java/com/zufar/onlinestore/user/api/UserService.java

```diff
@@ -15,6 +15,7 @@ public class UserService implements UserApi {
 
     private final SaveUserOperationPerformer saveUserOperationPerformer;
     private final SingleUserProvider singleUserProvider;
+    private final GetPasswordByEmail getEmail;
```

Comment:
that is a bad naming for class even though it encapsulates only single operation. Smth like PasswordRetrievalService would be so much better